docs/deploy/09-workload.md

```diff
@@ -34,28 +46,36 @@ The cluster now has an [Traefik configured with a TLS certificate](./09-secret-m
+   > In this moment your Ingress Controller (NGINX) is reading your ingress resource object configuration, updating its status, and creating a router to fulfill the new exposed workloads route. Take a look at this and notice that the address is set with the Internal Load Balancer IP from the configured subnet.
 
    ```bash
    kubectl get ingress aspnetapp-ingress -n a0008
    ```
 
-   > At this point, the route to the workload is established, SSL offloading configured, a network policy is in place to only allow Traefik to connect to your workload, and Traefik is configured to only accept requests from App Gateway.
+   > At this point, the route to the workload is established, SSL offloading configured, a network policy is in place to only allow NGINX to connect to your workload, and NGINX is configured to only accept requests from App Gateway.
+
+1. Check the Ingress KeyVault attachment is working properly
+
+   ```bash
+   kubectl get secret/keyvault-aspnetapp-ingress -n a0008
+   ```
 
 1. Test direct workload access from unauthorized network locations. *Optional.*
 
    > You should expect a `403` HTTP response from your ingress controller if you attempt to connect to it *without* going through the App Gateway. Likewise, if any workload other than the ingress controller attempts to reach the workload, the traffic will be denied via network policies.
 
    ```bash
-   kubectl run curl -n a0008 -i --tty --rm --image=mcr.microsoft.com/devcontainers/base --overrides='[{"op":"add","path":"/spec/containers/0/resources","value":{"limits":{"cpu":"200m","memory":"128Mi"}}},{"op":"add","path":"/spec/containers/0/securityContext","value":{"readOnlyRootFilesystem": true}}]' --override-type json  --env="DOMAIN_NAME=${DOMAIN_NAME_AKS_BASELINE}"
+   kubectl run curl -n a0008 -i --tty --rm --image=mcr.microsoft.com/devcontainers/base --overrides='[{"op":"add","path":"/spec/containers/0/resources","value":{"limits":{"cpu":"200m","memory":"128Mi"}}},{"op":"add","path":"/spec/containers/0/securityContext","value":{"readOnlyRootFilesystem": true}}]' --override-type json --env="DOMAIN_NAME=${DOMAIN_NAME_AKS_BASELINE}"
 
    # From within the open shell now running on a container inside your cluster
    curl -kI https://bu0001a0008-00.aks-ingress.$DOMAIN_NAME -w '%{remote_ip}\n'
    exit
    ```
 
-   > From this container shell, you could also try to directly access the workload via `curl -I http://<aspnetapp-service-cluster-ip>`. Instead of getting back a `200 OK`, you'll receive a network timeout because of the [`allow-only-ingress-to-workload` network policy](../../cluster-manifests/a0008/ingress-network-policy.yaml) that is in place.
+   > From this container shell, you could also try to directly access the workload via:
+   > - `curl -I http://bu0001a0008-00.aks-ingress.$DOMAIN_NAME -w '%{remote_ip}\n'`. Instead of `403` you are now getting back a `308 Permanent Redirect` and the location will be with the https protocol instead.
```

Comment:
```suggestion
   > - `curl -I http://bu0001a0008-00.aks-ingress.$DOMAIN_NAME -w '%{remote_ip}\n'`. Instead of `403` you are now getting back a `308 Permanent Redirect` and the new location uses the HTTPS protocol instead.
```